CHANGELOG.md

```diff
@@ -1,4 +1,15 @@
-## Unreleased
+## [0.0.5] - 27 September 2018
```

Comment:
As the version is presently `0.0.2`, `0.0.3-0.0.5` can be consolidated into an `0.0.3` release